### ISSUE TITLE
[Question] Behavior queues

### ISSUE THREAD

**Issue by maprietom** [volcano-sh/volcano#199]
I have a Kubernetes cluster with Volcano installed, but I don't understand how queues work. I create a queue and I launch jobs by Volcano until the cluster don't have resources free so I launch another job and Volcano stop randomly one job and last job is running instead of "Pending" waiting for resources free. Is normal this behavior?

**Comment by k82cn** [comment]
@maprietom , what's your configuration of scheduler? please try to disable `preempt` action :)

**Comment by maprietom** [comment]
Another question ...

I'm trying a scenario with 2 queue (2 and 3 weights respectively). I launched 2 jobs onto queue with weight 2, then I launched 1 job onto queue with weight 3 and then I launched another job onto queue with weight 3 but no resources free and this job wait for resources, I thought that Volcano abort a job in queue with weight 2 to break free resources and the job go running. Is it correct my idea?
